src/issue/screens/issue.screen.js

```diff
@@ -55,10 +60,21 @@ const mapDispatchToProps = dispatch =>
       postIssueComment,
       getIssueFromUrl,
       deleteIssueComment,
+      getIssueEvents,
     },
     dispatch
   );
 
+const compareCreatedAt = (a, b) => {
+  if (a.created_at < b.created_at) {
+    return -1;
+  } else if (a.created_at > b.created_at) {
+    return 1;
+  }
+
+  return 0;
+};
+
```

Comment:
How about `(a, b) => a.created_at - b.created_at`? So that we can omit this function.

Reply:
i think `created_at` is a date string, so i don't think we can `-` them.

Reply:
@brandly Yes, my mistake. 🙈